cardano-cli/src/Cardano/CLI/Shelley/Run.hs

```diff
@@ -26,5 +29,6 @@ runShelleyClientCommand cc =
 
 
 runShelleyKeyGenerate :: OutputFile -> ByteString -> ExceptT CliError IO ()
-runShelleyKeyGenerate fpath title =
-  liftIO . putStrLn $ "runShelleyKeyGenerate (" ++ show fpath ++ ") " ++ show title
+runShelleyKeyGenerate (OutputFile fpath) _title = do
+  kp <- liftIO $ shelleyGenKeyPair RegularShelleyKey
+  firstExceptT CardanoApiError (ExceptT (writeKeyPair fpath kp))
```

Comment:
You can use `liftIO` or `newExceptT` here

Reply:
I don't mind switching it to `newExceptT`, but:

```
-- | Constructor for computations in the ExceptT monad.
-- (The inverse of 'runExceptT').
newExceptT :: m (Either x a) -> ExceptT x m a
newExceptT =
  ExceptT
```

Reply:
Switched it to `newExceptT`.